ariadne_codegen/client_generators/client.py

```diff
@@ -262,7 +263,7 @@ def _generate_variables_assign(
         return generate_ann_assign(
             target=self._variables_dict_variable,
             annotation=generate_subscript(
-                generate_name("dict"),
+                generate_name("Dict"),
```

Comment:
We should use the constant as well
```suggestion
                generate_name(DICT),
```